haystack/components/rankers/transformers_similarity.py

```diff
@@ -14,6 +14,7 @@
 with LazyImport(message="Run 'pip install transformers[torch,sentencepiece]'") as torch_and_transformers_import:
```

Comment:
Should we update this message here?

Reply:
the `accelerate` library is automatically installed with `pip install transformers[torch,sentencepiece]` (see https://github.com/huggingface/transformers/blob/f7ef7cec6c6c162087421f36a17eabdbb223579d/setup.py#L262),
so I think the message is ok.

Reply:
Cool, didn't know